server/service/mdm.go

```diff
@@ -738,3 +738,110 @@ func (svc *Service) GetMDMCommandResults(ctx context.Context, commandUUID string
 	}
 	return results, nil
 }
+
+////////////////////////////////////////////////////////////////////////////////
+// GET /mdm/commands
+////////////////////////////////////////////////////////////////////////////////
+
+type listMDMCommandsRequest struct {
+	ListOptions fleet.ListOptions `url:"list_options"`
+}
+
+type listMDMCommandsResponse struct {
+	Results []*fleet.MDMCommand `json:"results"`
```

Comment:
I believe we always want to add the `Metadata` to paginated endpoints now?

Reply:
@mna sorry for bothering, I think didn't pay attention to the discussion about metadata. Could you help me understand what's the idea? should I mimic what we did for Activities?

1. check `ListOptions.IncludeMetadata`
2. if included, calculate and add a `fleet.PaginationMetadata` object to the response

Reply:
@roperzh Let me know if that's not clear, we can always hop on a quick call.

Reply:
thank you very much! super clear now. Believe it or not, this branch  doesn't have the list scripts endpoint lol. I might do this right after updating the branch.

Reply:
> this branch doesn't have the list scripts endpoint

:D Oh wow, it's missing so many things by now!